membership-common/src/test/resources/rest/plans/Credits.json

```diff
@@ -16,10 +16,10 @@
   "serviceActivationDate": "2024-06-13",
   "customerAcceptanceDate": "2024-06-28",
   "subscriptionStartDate": "2024-06-13",
-  "subscriptionEndDate": "2025-06-13",
+  "subscriptionEndDate": "2025-07-13",
```

Comment:
If this date needs to be in the future would it be worth setting the year to 2099?

Reply:
the tests shouldn't really be relying on the current date at all, I think a load of tests were added about a year ago that did.  Some of them I have changed over to use a hard coded date passed in but I must have missed this one.